src/modules/connection.js

```diff
@@ -95,8 +101,8 @@ class Connection {
    * Connect to AMQP and create channel
    * @return {Promise} A promise that resolve with an amqp.node channel object
    */
-  get() {
-    return this.getConnection().then(() => this.getChannel());
+  get(queue, prefetch) {
```

Comment:
Maybe we should accept here "options" instead of prefetch, in case we have more arguments we want to pass along